docs/integrations/introduction.rst

```diff
@@ -0,0 +1,41 @@
+Introduction to  Neptune Integrations
+=====================================
+
+.. image:: ../_static/images/overview/framework-logos.png
+   :target: ../_static/images/overview/framework-logos.png
+   :alt: selected integrations
+
+Neptune comes with 25+ integrations with Python libraries popular in machine learning, deep learning and reinforcement learning.
+
+How integrations work?
+----------------------
+Integrations are written using `Neptune-client <../python-api/introduction.html>`_ and provide a convenient way to jump-start working with Neptune and particular library. There is no need to integrate your code manually using neptune-client (it's easy though).
```

Comment:
I'd go with "and a library that you are using".

Reply:
changed.